scrapy/core/http2/stream.py

```diff
@@ -253,14 +255,9 @@ def receive_data(self, data: bytes, flow_controlled_length: int):
 
         if self._log_warnsize:
             self._reached_warnsize = True
-            warning_msg = 'Received more ({bytes}) bytes than download ' \
-                          + 'warn size ({warnsize}) in request {request}'
-            warning_args = {
-                'bytes': self._response['flow_controlled_size'],
-                'warnsize': self._download_warnsize,
-                'request': self._request
-            }
-            logger.warning(warning_msg.format(**warning_args))
+            warning_msg = f"Received more ({self._response['flow_controlled_size']}) bytes than download " \
```

Comment:
In Scrapy, we tend to prefer

```python
variable = ("string over "
            "multiple lines")
```

or, specially with longer strings,

```python
variable = (
    "string over "
    "multiple lines"
)
```

over

```python
variable = "string over " \
           + "multiple lines"
```